packages/@aws-cdk/aws-ecr/README.md

```diff
@@ -79,6 +79,26 @@ You can set tag immutability on images in our repository using the `imageTagMuta
 new ecr.Repository(this, 'Repo', { imageTagMutability: ecr.TagMutability.IMMUTABLE });
 ```
 
+### Encryption
+
+By default a Repository is server-side encrypted with AES-256 by AWS. However, you can use AWS managed KMS or customer managed KMS. When `encryptionKey` is set, the `encryption` must be `KMS` or empty.
```

Comment:
Please add a link to the ECR developer guide where people can read in more detail about this feature: https://docs.aws.amazon.com/AmazonECR/latest/userguide/encryption-at-rest.html 

Lifting some sentences from that doc and adding more explanation for clarity here: 

```suggestion
By default, Amazon ECR uses server-side encryption with Amazon S3-managed encryption keys which encrypts your data at rest using an AES-256 encryption algorithm. For more control over the encryption for your Amazon ECR repositories, you can use server-side encryption with KMS keys stored in AWS Key Management Service (AWS KMS). When you use AWS KMS to encrypt your data, you can either use the default AWS managed key, which is managed by Amazon ECR, by specifying `RepositoryEncryption.KMS` in the `encryption` property. Or specify your own customer managed KMS key, by specifying the `encryptionKey` property. 

When `encryptionKey` is set, the `encryption` property must be `KMS` or empty.
```